### PR TITLE
Methods of organizationManager were fixed

### PR DESCRIPTION
**Pull request by fuga02**

### CODE REVIEW

File: src/Marketplace.Services.Organizations/Managers/OrganizationManager.cs [hunk 1]
```diff
@@ -44,6 +105,29 @@ public async Task<OrganizationModel> GetById(Guid organizationId)
...
+            Description = organization.Description,
+            Logo = organization.Logo,
+            Contact = organization.Contact,
+            Users = organization.Users,
+            Addresses = organization.Addresses!
+                .Select(address => new OrganizationAddressModel()
+                    { Id = address.Id, Address = address.Address })
+                .ToList()
+        };
+
```
Comment (javlonfattoev) [inline src/Marketplace.Services.Organizations/Managers/OrganizationManager.cs:131]: organization entityni modelga parse qiladigan method yozing shunda takrorlanmaydi kod

File: src/Marketplace.Services.Organizations/Managers/OrganizationManager.cs [hunk 2]
```diff
@@ -20,22 +21,82 @@ public class OrganizationManager
...
 
-	}
+        organization.Addresses = organizationModel.Addresses!.Select(model => new OrganizationAddress()
+        {
+            Id = Guid.NewGuid(),
+            OrganizationId = organization.Id,
+            Organization = organization,
+            Address = model.Address
+        }).ToList();
+        _organizationsDbContext.Organizations.Add(organization);
```
Comment (javlonfattoev) [inline src/Marketplace.Services.Organizations/Managers/OrganizationManager.cs:62]: modelda address null bolmasa saqlasin

**Review by javlonfattoev — CHANGES_REQUESTED** [review]

File: src/Marketplace.Services.Organizations/Managers/OrganizationManager.cs (hunk 1 above)
Comment (fuga02) [reply src/Marketplace.Services.Organizations/Managers/OrganizationManager.cs]: Added

File: src/Marketplace.Services.Organizations/Managers/OrganizationManager.cs (hunk 2 above)
Comment (fuga02) [reply src/Marketplace.Services.Organizations/Managers/OrganizationManager.cs]: All done

**Pull request merged**